embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/fakes/injection/FakeEssentialServiceModule.kt

```diff
@@ -52,6 +55,10 @@ internal class FakeEssentialServiceModule(
     override val networkConnectivityService: NetworkConnectivityService = NoOpNetworkConnectivityService(),
     override val pendingApiCallsSender: PendingApiCallsSender = FakePendingApiCallsSender(),
     override val urlBuilder: ApiUrlBuilder = FakeApiUrlBuilder(),
+    override val hostedSdkVersionInfo: HostedSdkVersionInfo = HostedSdkVersionInfo(
+        FakePreferenceService(),
+        mockk()
```

Comment:
We can pass `InternalEmbraceLogger()` instead of mock. Same for a couple of other tests

Reply:
Actually is not required after removing `logDeveloper`.